cmd/cloudx/project/get_oauth2_config.go

```diff
@@ -43,7 +43,11 @@ $ ory get oauth2-config ecaaa3cb-0730-4ee8-a6df-9553cdfeef89 --format json
 				return err
 			}
 
-			project, err := h.GetProject(args[0])
+			var id string
+			if id, err = getSelectedProjectId(h, args); err != nil {
```

Comment:
We would usually write this:

```
id, err := getSelectedProjectId(h, args);
if err != nil {
 ...
}
```

When you have multiple return arguments, it shortens the code:

```
a, b, c, err := getSelectedProjectId(h, args);
if err != nil {
 ...
}
```

```
var a a
var b b
car c c
if a, b, c, err = getSelectedProjectId(h, args);err != nil {
 ...
}
```